sourcemash/api/items.py

```diff
@@ -75,9 +80,12 @@ def put(self, id):
         # Check vote column of user_items table
         try:
             user_item = UserItem.query.filter_by(user=current_user, item=item).one()
+            user_item.last_modified = datetime.utcnow()
+            db.session.commit()
```

Comment:
You should be able to do just one commit at the end of the API call, rather than after each change.